woocommerce/woocommerce-gutenberg-products-block
# Render HTML in description of active payment integrations

Currently, the descriptions of active payment integrations do not render HTML when viewing the Checkout block in the editor. This PR aims to solve this problem.

Fixes #7312 

### Screenshots

<table>
<tr>
<td>Before:
<br><br>

![7312-before](https://user-images.githubusercontent.com/3323310/194226310-aef0c678-309d-40dd-bfb7-8a39dff5448e.png)
</td>
<td>After:
<br><br>

![7312-after](https://user-images.githubusercontent.com/3323310/194226296-aadc639f-a01c-417c-92c9-214c014b5ae2.png)
</td>
</tr>
</table>

### Testing

1. Install the [WooCommerce Stripe Payment Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/).
2. Head over to `/wp-admin/admin.php?page=wc-settings&tab=checkout`.
3. Activate various Stripe payment methods, e.g. Stripe SEPA Direct Debit.
4. Head over to the Checkout block in the editor.
5. Click on the Payments Options inner block.
6. Verify that all descriptions of active payment integrations render HTML.


### WooCommerce Visibility


### Changelog

> Fix: Render HTML in description of active payment integrations.
